defaults/main.yml

```diff
@@ -78,6 +78,11 @@ bind9_packages:
     - dnsutils
     - haveged
 
+# Directory for bind9 files templates
+bind9_template_named_conf: bind
+## you can set your own templates, for example with
```

Comment:
I think this won't pass linting. Please make normal comments. Start with a capital letter.

Reply:
Ok, thanks for the tip.